modules/ppcp-wc-gateway/src/Gateway/PayUponInvoice/PayUponInvoice.php

```diff
@@ -246,6 +246,10 @@ function ( WC_Order $wc_order, string $order_id ) {
 					if ( $breakdown ) {
 						$wc_order->update_meta_data( PayPalGateway::FEES_META_KEY, $breakdown->to_array() );
 						$wc_order->save_meta_data();
+						$paypal_fee = $breakdown->paypal_fee();
+						if ( $paypal_fee ) {
+							update_post_meta( $wc_order->get_id(), 'PayPal Transaction Key', $paypal_fee->value() );
```

Comment:
I thought we need to use WC order's `update_meta_data` instead of the WC function?

Reply:
hm, I thought it depends only on name? So if you use the same name without `_` prefix, then it will show it.

Reply:
You´re rigth, I just added it here also in `woocommerce_paypal_payments_order_captured` filter, thanks.